elementary/monitor/data_monitoring/schema.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, validator
+
+from elementary.utils.time import DATETIME_FORMAT, convert_local_time_to_timezone
+
+
+class DataMonitoringFilter(BaseModel):
+    invocation_id: Optional[str] = None
+    invocation_time: Optional[str] = None
+    last_invocation: Optional[bool] = False
+
+    @validator("invocation_time", pre=True)
+    def format_invocation_time(cls, invocation_time):
+        if invocation_time:
+            invocation_datetime = convert_local_time_to_timezone(
+                datetime.fromisoformat(invocation_time)
+            )
```

Comment:
When documenting this feature - it's worth noting that the invocation_time is expected in local time

Reply:
Will add! (the documentation is already merged - was part of the old docs branch 🙂 )